Bucardo/object_types/sync.md

```diff
@@ -3,21 +3,17 @@ title: Sync
 ---
 
 Bucardo replication events happens in **syncs**, which replicate one or more
-tables and come in one of three types:
-
--   [fullcopy](/Bucardo/object_types/fullcopy)
--   [pushdelta](/Bucardo/object_types/pushdelta)
--   [swap](/Bucardo/object_types/swap)
+tables.
 
 To add a new sync, run:
 
-    bucardo add sync <syncname> source=xx targetdb=yy targetgroup=zz type=<type> tables=x,y,z
```

Comment:
I'm a little uncertain about removing `tables=x,y,z` since that is still valid as per `bucardo help add sync`. I agree it shouldn't be shown as the primary thing to do here, but maybe a footnote to explain it's shorthand for adding a relgroup?

Reply:
It is explained below this.